js/blocks/WidgetBlocks.js

```diff
@@ -64,8 +64,19 @@
 
 /* exported setupWidgetBlocks */
 
+/**
+ * Sets up blocks related to sound envelope (ADSR) for the given activity.
```

Comment:
This comment is not in the right place. It belongs with the envelope block.

Reply:
hi walter, but isn't function setupWidgetBlocks(activity) helping us to set up blocks related to sound envelope (ADSR) and EnvelopeBlock represents a block for controlling sound envelope (ADSR)?

Reply:
SetupWidgetBlocks is the wrapper for all of the block class definitions. Not just the Envelope Block.

Reply:
Acknowledged. The necessary changes have been implemented.